### ISSUE TITLE
Forking the prediction API (backend)

### ISSUE THREAD

**Issue by David-Rey** [VIP-LES/EosGround#26]
Blocked by #25 

- Fork API
- switch the front end to use the locally API
- Test that it works

**Comment by thomasmholder** [comment]
https://github.com/projecthorus/tawhiri
https://github.com/projecthorus/tawhiri-downloader-container/

**Comment by thomasmholder** [comment]
@DarylDohner @David-Rey The prediction backend can run without the frontend, does this need to be blocked by #25 ?

**Comment by DarylDohner** [comment]
Yes b/c one of the requirements is switching the frontend to pull from the local API and another is to test them together
